File: admin/src/app/models/update-requests/category-detail-update-request.ts
import { AutoMap } from "@automapper/classes";
import { BaseUpdateRequest } from "../bases";


export class CategoryDetailUpdateRequest extends BaseUpdateRequest {
    @AutoMap()
    id?: number;
    @AutoMap()
    langId: string;
    @AutoMap()
    name: string;
    @AutoMap()
    description: string;
}

File: App.Models/DTOs/CategoryDetails/CategoryDetailUpdateRequest.cs
using App.Models.DTOs.UpdateRquests;
using App.Common.Model.DTOs;

namespace App.Models.DTOs.CategoryDetails
{
    public class CategoryDetailUpdateRequest : BaseUpdateRequest<int>
    {
        public int CategoryId { get; set; }
        public string LangId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
    }
}
